JavaScript/Algorithms/BoyerMooreAlgorithm.js

```diff
@@ -0,0 +1,100 @@
+$(document).ready(() => {

```

Comment:
I don't think we need this jQuery method in this program, moreover the program won't be executed in many online Node.js IDE due to this.

Reply:
Okay I'll update the respective.